src/domains/Form/routes/FormAnswers/FormAnswers.js

```diff
@@ -1,19 +1,72 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ActionsFunctionsContext from '../../../../context/ActionsFunctions/ActionsFunctionsContext'
 import { TranslationContext } from '../../../../context/Translation'
 import TypeformConfigurationContext from '../../../../context/TypeformConfigurationContext'
 import FirebaseContext from '../../../../context/Firebase/FirebaseContext'
 import ResponseList from '../../../../domains/Response/components/ResponseList'
+import useFunctions from '../../../../hooks/useFunctions'
+import { COLLECTIONS } from '../../../../constants'
+import { useCollectionData } from 'react-firebase-hooks/firestore'
+import { Row, Col } from '@qonsoll/react-design'
+import { ResponseTable } from '../../../Response/components'
+import { message } from 'antd'
 
 function FormAnswers(props) {
   const { actions = {}, id, translate, firebase, configurations } = props
 
+  // [COMPONENT STATE HOOKS]
+  const [userAnswers, setUserAnswers] = useState([])
+  const [userAnswersLoading, setUserAnswersLoading] = useState(false)
+  // [CUSTOM HOOKS]
+  const { getCollectionRef } = useFunctions(firebase)
+
+  // [ADDITIONAL HOOKS]
+  const [userAnswerGroup, userAnswerGroupLoading] = useCollectionData(
+    getCollectionRef(COLLECTIONS.USER_ANSWERS_GROUP).where('formId', '==', id)
+  )
+
+  // [CLEAN FUNCTIONS]
+  const onListItemClick = async (user, date) => {
+    try {
+      setUserAnswersLoading(true)
+      const answers = await getCollectionRef(COLLECTIONS.ANSWERS)
+        .where('user', '==', user)
+        .where('date', '==', date)
+        .where('formId', '==', id)
+        .get()
+      const answersData = answers.docs.map((item, index) => ({
```

Comment:
I think u should add checking when there is no answers to prevent error map of undefined something like answers?.docs?.map and also for fields below item?.data()?.questionTitle and so on